examples/src/utils/MeasurementUi.ts

```diff
@@ -1,7 +1,7 @@
+import { THREE } from '@cognite/reveal';
+import dat from "dat.gui";
 import { Cognite3DViewer } from "@cognite/reveal";
 import { MeasurementTool } from "@cognite/reveal/tools";
-import dat from "dat.gui";
-import { THREE } from '@cognite/reveal';
 
 export class MeasurementUi {
   private readonly _viewer: Cognite3DViewer;
```

Comment:
Just curious - do we have a convention on the order of imports in places like these?

Reply:
Just feel like external import should be first. No reason really, we should probably just add some lint rule for this.